src/AuthorizedSocketConnection.js

```diff
@@ -98,24 +98,22 @@ export default class AuthorizedSocketConnection<TContext, TCredentials> {
     { id, query, variables }: Subscription,
     cb?: Function,
   ) => {
-    if (
-      this.config.maxSubscriptionsPerConnection != null &&
-      this.subscriptions.size >= this.config.maxSubscriptionsPerConnection
-    ) {
-      this.log('error', 'Max Subscription limit reached', {
-        maxSubscriptionsPerConnection: this.config
-          .maxSubscriptionsPerConnection,
-      });
-      this.emitError({
-        code: 'subscribe_failed.subscription_limit',
-      });
-      acknowledge(cb);
-      this.config.socket.disconnect();
-      return;
-    }
-
     let result;
     try {
+      if (
+        this.config.maxSubscriptionsPerConnection != null &&
+        this.subscriptions.size >= this.config.maxSubscriptionsPerConnection
+      ) {
+        this.log('error', 'Max Subscription limit reached', {
```

Comment:
do we need to log this as a server error? _we_ haven't done anything wrong here, have we?

Reply:
good point